src/ts/components/core/Highlight.tsx

```diff
@@ -9,7 +9,7 @@ interface Props extends DashBaseProps, TextProps {
     /** Key of `theme.colors` or any valid CSS color, passed to `Mark` component `color` prop, `yellow` by default */
     color?: MantineColor | string;
     /** Styles applied to `mark` elements */
-    highlightStyles?: React.CSSProperties;
```

Comment:
Oh wow, I would not have guessed that `React.CSSProperties` blows up into a whole CSS reference manual! (cc @t4rk1n I wonder if we could simplify this particular case at the codegen stage?)

This change looks great, I'd just suggest adding a note that these are CSS but property names are camelCased - that's pretty universal across Dash but still trips people up.